src/webgl/p5.RendererGL.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 var p5 = require('../core/core');
-var shader = require('./shader');
+var builtInShaders = require('./shader');
```

Comment:
hey, just a reminder we should change this to defaultShaders.js or shaderBuilder.js